fastify.js

```diff
@@ -40,14 +40,17 @@ function build (options) {
 
   var log
   if (isValidLogger(options.logger)) {
-    log = loggerUtils.createLogger({ logger: options.logger, serializers: loggerUtils.serializers })
+    log = loggerUtils.createLogger({
+      logger: options.logger,
+      serializers: Object.assign({}, loggerUtils.serializers, options.logger.serializers || {})
```

Comment:
This line works only with `pino`: if you are using another logger `serializers` property is not set. I like to see this in the doc. Could you drop a line for this?

Reply:
Done, thanks for the fast review!

Reply:
This is not needed `options.logger.serializers || {}` you can do

```js
 serializers: Object.assign({}, loggerUtils.serializers, options.logger.serializers)
```

Reply:
Fixed!